persistence/persistence-api/src/main/java/org/cbioportal/persistence/CacheDisableSetting.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.persistence;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class CacheDisableSetting {
```

Comment:
is there a reason we introduced `inverse`/`booleanValue` as additional data members instead of just using `setting`?

Reply:
if we have to keep to additional data members `inverse` and `booleanValue`, I would rename them to something for descriptive (especially `inverse`)

Reply:
Agreed ^ having the additional `inverse` member is a bit confusing.

Reply:
I've changed the setting from disabled to enabled .. so I will remove the inverse member
The booleanValue is to save the backend the time of many times converting string "false" into a boolean false value (or "true" to true)